queries/swift/highlights.scm

```diff
@@ -1,3 +1,6 @@
+[ "." ";" ":" "," "->" ] @punctuation.delimiter
```

Comment:
`->` is apparently used to indicate return type in functions. `->` is an operator in Rust. So let's switch to that variant